README.md

```diff
@@ -1,12 +1,11 @@
-packager
-========
+# GOV.UK packaging
 
-This repository contains some scripts to aid the building of debian source
-packages (.dsc) files for submission to Launchpad or another build server for
-building into binary debian packages (.deb)
+This repo creates packages for GOV.UK.
 
-Usage
------
+## fpm
 
-See the ["Getting Started" documentation at
-`doc/getting_started.md`](doc/getting_started.md).
+This is the way we create packages now. It is easier than using debuild.
```

Comment:
I always found it tough to build packages, probably because I'm not very familiar with (archaic? :trollface:) Debian conventions. fpm is significantly less work for me (and I suspect lots of other people on GOV.UK).

For example, the fact that the Debian changelog file has to exactly match your GPG ID (including comment) isn't something I could find out easily and caused me lots of pain: https://github.com/alphagov/packager/blob/cb53ae872ad1e7a5ae49b3925b3d2a50ed3282f7/pkg/grafana/debian/changelog#L17